Keyboards/KeyboardsBase/InterfaceVariables.swift

```diff
@@ -26,7 +26,7 @@ var keyWidth = CGFloat(0)
 var letterKeyWidth = CGFloat(0)
 var numSymKeyWidth = CGFloat(0)
 var isFirstKeyboardLoad = false
-
+var disableAccentCharacters = false;
```

Comment:
Oh interesting.. :thinking: just learned that semicolons are also optional in Swift :astonished: 
Not 100% on how y'all have been doing this, but we use [SwiftLint](https://github.com/realm/SwiftLint) for linting? Do we enforce linting? 

Total nit hehe - the above trailing semicolon simply caught my eye, and led me to searching a bit about SwiftLint :laughing: Appears like the community seems to prefer removing trailing semicolons? Not sure, but made me think about linting for us.

NOTE: no need to set any of this up in this PR :laughing: just commenting as something for us to think through :+1: 

CC: @andrewtavis @SaurabhJamadagni

Reply:
Ya I'd say we don't need the semicolons as well, thanks @wkyoshida! Maybe we can get ourselves one of them fancy PR linting checks 🤠 Should we make an issue for it?